apps/ui/src/Components/community-list/community-list.tsx

```diff
@@ -0,0 +1,53 @@
+import dayjs from 'dayjs';
+import React from 'react';
+import { isEmpty } from 'lodash';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import { Card, CardContent, Typography } from '@mui/material';
+
+import { Community } from '@communecar/types';
+
+import { RideCard } from '../ride-card/ride-card';
+
+dayjs.extend(relativeTime);
+
+type CommunityWithRides = Community & { rides: any[] };
+
+interface CommunityListProps {
+  communities: CommunityWithRides[];
+}
```

Comment:
Put these in a types file

Reply:
I left the props interface but moved the other type to types file